.travis.yml

```diff
@@ -9,7 +9,7 @@ python:
     - 3.3
     - 3.4
 install:
-    - travis_retry pip install BTrees ZConfig manuel persistent six transaction zc.lockfile zdaemon zope.interface zope.testing zope.testrunner
+    - travis_retry pip install BTrees ZConfig manuel persistent six transaction zc.lockfile zdaemon zodbpickle zope.interface zope.testing zope.testrunner
```

Comment:
I agree that's a lot of repetition there, but it fits the pattern of what was done before. Perhaps we can clean up .travis.yml in a subsequent PR (enable caching and so forth)?